foos/config_base.py

```diff
@@ -61,6 +61,11 @@
 btn_sleep_time = 0.01
 btn_long_duration = 0.4
 
+#ir barrier configuration
+ir_pin_a = 24
```

Comment:
can do something like `ir_pin_a = (24, 'black')` or even better `ir_pins = {24: 'black', 23: 'yellow'}` and avoid making changes to the code in the plugin should we need to replace the pins.